bittensor/commands/unstake.py

```diff
@@ -349,10 +350,27 @@ def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
         # Parse from strings
         netuid = cli.config.netuid
 
-        success, message = subtensor.revoke_children(
+        # Get current children
+        current_children = subtensor.get_children_info(netuid=netuid)
+        if (
```

Comment:
More of a suggestion; it might be better to write this if condition as:
if not current_children or not current_children.get(cli.config.hotkey, False):